shared/src/main/java/com/couchbase/lite/DocumentReplication.java

```diff
@@ -24,31 +24,20 @@
  * Document replicated update of a replicator.
  */
 public final class DocumentReplication {
-
     //---------------------------------------------
     // member variables
     //---------------------------------------------
     private final Replicator replicator;
-    private boolean isDeleted = false;
-    private boolean isAccessRemoved = false;
     private boolean pushing = false;
-    private String docId = "";
-    private String revId = "";
-    private int flags;
-    private C4Error error;
-    private boolean trans;
+    private ReplicatedDocument[] documents;
```

Comment:
I have just noticed this. Can you change the type to List\<ReplicatedDocument\> documents?

For reference, we have used List<String>documentIDs in DatabaseChange class so I think we should do the same here for consistency.